### DISCUSSION TITLE
Development stopped

### DISCUSSION THREAD

**Discussion by tkowar** [mealie-recipes/mealie#2466] [General]
Are there any plans for further development? It seems that development has stopped.

**Comment by michael-genson** [comment]
Which version of Mealie are you looking at? [Just today there have been several merged PRs](https://github.com/hay-kot/mealie/pulls?q=is%3Apr+is%3Aclosed). If you're looking at the old version 0.5.6, which is outdated by a few years, you're not going to see any changes there.

**Reply by tkowar** [reply]
You are right, i didn't see the PRs 🤦‍♂️  I'm running the (i believe) latest `v1.0.0beta-5`. My concerns stem from the fact, that there have been regular frequent releases, but the current version was not updated since 8 months.

**Reply by hay-kot** [reply]
@tkowar that's fair. I've been doing a rather large move across the country and time has gotten away from me. Once all of my belongings arrive at my new house and I get my office a bit more settled, I'm planning knocking out the last few items for the v1 release. Shooting for end of a August to have something new out.

**Reply by tkowar** [reply]
@hay-kot It was not my intention to sound unthankful - it's in fact a great project and i want to see it thrive! Thanks for your response and good luck with moving 🙂 The question is (more than) answered, i'm closing the discussion.
